wiki/Mapping_and_Modding_Timeline/en.md

```diff
@@ -0,0 +1,453 @@
+    - Some concerns at the time about the QAT included:
+        - Little QAT engagement in controversial maps.
+        - Limited accessibility for new users to enter the team, causing stagnation.
+        - The role of the QAT in the mapping community was unclear.<!-- https://osu.ppy.sh/community/forums/topics/842601 -->
+
+**2749 beatmaps** were ranked in 2018. This year's impressive growth can be largely attributed to the QAT's discontinued Quality Assurance role and higher initiative from Beatmap Nominators.
+
+## 2019
+
+### January
+
+- **2019-01-29:** A proposal for QAT reform was created as a conglomeration of other users' proposals.
+    - The [Nomination Assessment Team (*NAT*)](/wiki/People/The_Team/Nomination_Assessment_Team) was introduced through this proposal, and was implemented later this year. <!-- https://osu.ppy.sh/community/forums/topics/862002 -->
+
+### March
+
+- **2019-03-01:** The [Mappers' Guild](/wiki/Mappers_Guild) transitioned from application-based entry to automatic entry for anyone with 3 ranked maps.
+    - [Featured Artist](/wiki/Featured_Artists) beatmaps became more frequently created and ranked.
+    - `#mappers-guild` channel was created in the [osu!dev Discord server](/wiki/osu!dev_Discord_server).
+- **2019-03-13:** Rule requiring consecutive nominations to be set at least 24 hours apart from each other was removed.
+    - This rule was a major complaint from mappers and Beatmap Nominators because it felt like it was unnecessarily wasting everyone's time. If an issue was found after the first nomination, the second nominator would often avoid pointing it out to avoid an extra 24 hour wait.
+    - Along with this change, Beatmap Nominators could veto qualified beatmaps. The rule originally existed because Beatmap Nominators needed 24 hours to potentially apply vetoes to maps with one nomination.<!-- https://osu.ppy.sh/community/forums/posts/6885033 -->
+
+### May
+
+- **2019-05-05:** The BN/QAT restructure was finalized.
+    - The Quality Assurance Team became the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team).
+    - Beatmap Nominators became responsible for disqualifications and veto mediation.
+    - The [Beatmap Nominator website](https://bn.mappersguild.com/) was published to automate many BN/NAT tasks. <!-- https://osu.ppy.sh/home/news/2019-05-05-bn-and-qat-restructure -->
+
+### August
+
+- **2019-08-05:** [Naxess](https://osu.ppy.sh/users/8129817) released the Mapset Verifier <!-- TODO: stub --> tool, automating a significant portion of beatmap checking processes for Beatmap Nominators.
+- **2019-08-06:** Beatmap Nominators became able to participate in BN application evaluations.
+    - Their input was used to determine applicants and to judge potential new NAT additions. <!-- https://github.com/pishifat/qat/commit/a8245a4c6fefcfe9368ffc1eee7e13ad6f8bb6af -->
```

Comment:
Hmm, but BNs' evaluations don't play a very large role in this, so saying that they determine applicants (? applicants what) seems misleading